### ISSUE TITLE
[Accessibility] Improve reading of toots with VoiceOver

### ISSUE THREAD

**Issue by MarcoZehe** [DagAgren/toot#123]
Currently, when touching a toot, VoiceOver does a pretty good job at reading it already. The only thing missing is the author name.

The most logical and efficient order would be:

Author name, Toot text (or CW bit if content warning is present and not revealed yet, toot text if it is), time stamp.

The latter two are almost already there. Only if a content warning is present, the toot text gets automatically read already, along with the "Tap to reveal" prompt. The toot text shold be suppressed in this case if not yet revealed, because I might also decide to not read a toot.

**Comment by chucker** [comment]
It also doesn't read relative time stamps right:

* it reads `5h` as "five h" and `1d` as "one d", rather than "five hours" and "one day"
* it reads `20s` as "twenties" (i.e., it thinks it's plural)
* oddly, it reads `1m` as "one meter" (so it does infer it's a unit?)

**Comment by DagAgren** [comment]
Tried to work out a first version of this now. Hopefully I got enough information into the new label, but file issues later if stuff is missing.
